src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -17,9 +19,9 @@ library ConstantsLib {
     /// @notice Initial rate at target per second (scaled by WAD).
     int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
 
-    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
-    int256 public constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
+    /// @notice Mininimum rate at target per second (scaled by WAD) (0.025% APR).
```

Comment:
```suggestion
    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
```
the variable is the rate **at target**, so it's still 0.1% right ?

Reply:
I actually made this change so that values in both comments are coherent
But they indeed don't correspond to the rate at target, but instead to the APR corresponding to the rate at target in the worst condition (-100% error for min, 100% error for max)

Reply:
Ah yes they are coherent, my bad. I still think that in the comment we should put the most readable thing. And since those variables are describing the rates at target, I'm in favor of writing 0.1% / 250%